src/lib/exec.ts

```diff
@@ -10,6 +15,7 @@ const spawnPromised = async (cmd: string, args: string[], opts: SpawnOptionsWith
     const childProcess = spawn(cmd, args, {
         ...opts,
         stdio: process.env.APIFY_NO_LOGS_IN_TESTS ? 'ignore' : 'inherit',
+        ...process.platform === 'win32' ? windowsOptions : {},
```

Comment:
maybe put this into parens before you spread?

Reply:
Is there a need for that 👀. I mean I'll gladly do it, just curious

Reply:
i would personally expect the code you have there to not work at all, having parens there removes the doubt :D